lib/components/artist/articles/index.js

```diff
@@ -29,7 +29,7 @@ class Articles extends React.Component {
 
 const styles = StyleSheet.create({
   heading: {
-    marginBottom: 20,
+    marginBottom: 12,
```

Comment:
I really dislike having to resort to such calculations. Was there no way to get rid of the whatever else was gobbling up 8 pt on the other side?